### PR TITLE
 fix incident / indicator field in layouts and mappers

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/29841

## Description
remove/fix incident/indicator fields that do not exist in content from layouts.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json [hunk 1]
```diff
@@ -158,7 +158,7 @@
                             },
                             {
                                 "endCol": 2,
-                                "fieldId": "signatureoriginalname",
+                                "fieldId": "signature",
```
Comment (guykeller) [inline Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json:161]: On some cases you changed one fieldId to another - why?

Comment (ChanochShayner) [reply Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json:161]: After consulting with @yaakovi we decided that in cases where there are incident fields that belong to the pack and it does not appear in the layout, but there is a field in the layout that is similar to it (and did not exist in the content), Apparently this is a mistake in the name of the field and the author's reference is to the field that exists in the pack.

**Comment by yaakovi** [comment]
@guykeller what is the status here?

**Comment by guykeller** [comment]
@ChanochShayner please fix the build here

**Review by guykeller — APPROVED** [review]

**Pull request merged**